File: IpAddressJava/IpDisplay.java
public class IpDisplay {
    
    public static void displayResults(int[] octets) {
        String decimal = IpConverter.octetsToDecimal(octets);
        String binary = IpConverter.octetsToBinary(octets);
        String hex = IpConverter.octetsToHex(octets);
        
        System.out.println("Output:");
        System.out.println("Decimal: " + decimal);
        System.out.println("Binary : " + binary);
        System.out.println("Hex : " + hex);
    }
    
    public static void displayError(String message) {
        System.out.println("Error: " + message);
    }
}

File: IpAddressPython/ip_display.py
from ip_converter import octets_to_decimal, octets_to_binary, octets_to_hex

def display_results(octets):
    decimal = octets_to_decimal(octets)
    binary = octets_to_binary(octets)
    hex_str = octets_to_hex(octets)
    
    print("Output:")
    print(f"Decimal: {decimal}")
    print(f"Binary : {binary}")
    print(f"Hex : {hex_str}")

def display_error(message):
    print(f"Error: {message}")
